alexaContextProperties.js

```diff
@@ -65,6 +65,12 @@ AlexaContextProperties.prototype.powerLevelStateProperty = function (state) {
  * @param {*} state 
  */
 AlexaContextProperties.prototype.colorStateProperty = function (state) {
+  // color item may receive brightness value
+  if (!isNaN(state)) {
+    var num = parseInt(state);
+    state = '0,0,' + num;
+  }
+
```

Comment:
One of the changes I submitted was to use the latest openhab state opposed to the original requested state in Alexa response messages. If @digitaldan is fine with that, this change wouldn't be necessary.

c7e8a5a50b8f5ab936bee7f5d30b2ef8d35d2cbf